WP_Auth0.php

```diff
@@ -483,6 +480,33 @@ function wp_auth0_db_check_update() {
  * Core WP hooks
  */
 
+function wp_auth0_export_settings_admin_action() {
+	$options         = WP_Auth0_Options::Instance();
+	$import_settings = new WP_Auth0_Import_Settings( $options );
```

Comment:
Why is this not `$export_settings` and `WP_Auth0_Export_Settings` ?  I wonder if the WP class should be called `Import_Export_Settings` if it can be used for both?

Reply:
Actually, it made more sense to move the export functionality out of the import class.